### PR TITLE
Dothraki Cheat Sheet: Update aliases per #2967

### PR DESCRIPTION
**Pull request by varundey**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Provide an overview of the changes this pull request introduces.

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (@mention interested parties)

@GuiltyDolphin

---

Instant Answer Page: https://duck.co/ia/view/dothraki_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @varundey

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/language/dothraki.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "sourceUrl" : "http://www.dothraki.org/"
     },
     "aliases": [
-        "dothraki language", "dothraki phrasebook"
+         "dothraki phrasebook"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/dothraki.json]: Lets get this indented to a multiple of 4.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Dothraki Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/dothraki_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Dothraki Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/dothraki_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Dothraki Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/dothraki_cheat_sheet_mobile.png)

**Comment by GuiltyDolphin** [comment]
@varundey Thanks! :dolphin:

**Pull request merged**